### ISSUE TITLE
Command or Shift Key on a Mac shows the toolbar

### ISSUE THREAD

**Issue by hairydalek** [rawles/teletext-editor#53]
Referencing the lack of a save image, I tried taking a screen shot of the work area in Safari instead.

1 - Open or create a new image in Edit.tf

2 - Hide the toolbar using Escape+0

3 - The key combination for a screen shot on a Mac is Command+Shift+3 (or 4 if you was to capture just a window or a portion of the screen).

4 >>The tool bar and cursor reappear. Further testing show that they do pressing either the Command key or the Shift key.

Yes, I know I have graphic export options in Firefox! I just default to Safari. I have  Firefox bug too.

**Comment by rawles** [comment]
Hi. Are you asking that a change is made so that the modifier keys (Shift, Ctrl), etc, don't cause the tool bar and cursor to reappear?

**Comment by hairydalek** [comment]
Yes. If you take a screen shot, you don't expect things to change during the process. If it's necessary for those keys to restore UI elements, then a way to "lock" the UI for screen capture purposes would be a good idea.

**Comment by rawles** [comment]
I just added in a fix so that modifier key events are ignored. This means that they won't unhide the status bar. I tried to do a screenshot on my (El Capitan) Mac and it seemed to work in the way you want.
